cmd/build

```diff
@@ -99,6 +102,11 @@ elif [ "$1" == force ]; then
 elif [ "$1" == inline ]; then
     inline="inline"
     shift
+elif [ "$1" == tar ]; then
```

Comment:
"tar" is too specific (it's a mechanism, not a function) -- maybe "save"?  "export"?  What is this *doing*, not how does it do it...

Reply:
I went with compress